contrib/mesos/pkg/executor/executor.go

```diff
@@ -467,7 +473,7 @@ func (k *KubernetesExecutor) launchTask(driver bindings.ExecutorDriver, taskId s
 	task.podName = podFullName
 	k.pods[podFullName] = pod
 
-	// Send the pod updates to the channel.
+	// Send the pod update to the kubelet via the pod config update channel.
```

Comment:
`// send the latest snapshot of the set of pods to the kubelet via the pod update channel`